docker/scripts/dump-env.sh

```
@@ -0,0 +1,137 @@
+  echo "   Scanning $CONFIG_DIR for env() calls..."
+  # Find all env() calls in config files and extract the variable names
+  # Matches: env('VAR_NAME'), env("VAR_NAME"), env('VAR_NAME', default), etc.
+  KNOWN_VARS=$(find "$CONFIG_DIR" -name "*.php" -type f -exec grep -oE "env\(['\"]([A-Z0-9_]+)['\"]" {} \; | sed "s/env(['\"]//g" | sed "s/['\"]//g" | sort -u)
+  var_count=$(echo "$KNOWN_VARS" | wc -w)
+  echo "   Found $var_count unique environment variables in config files"
+else
+  echo "⚠️  WARNING: Config directory not found at $CONFIG_DIR"
+fi
+
+# Create or truncate the .env file
+>"$ENV_FILE"
```

Shellcheck (0.11.0)

[warning] This redirection doesn't have a command. Move to its command (or use 'true' as no-op).

(SC2188)